### DISCUSSION TITLE
Can't join server

### DISCUSSION THREAD

**Discussion by Katherite** [FabricMC/fabric-api#2863] [Player support] [answered]
I was using fabric 1.18.2 and joining a 1.18.2 fabric server and then this happened.
![image](https://user-images.githubusercontent.com/123320209/213930471-c32a26f2-4ef4-49f7-a56b-6f92e143ab2f.png)

**Comment by warjort** [comment, marked as answer]
Looks like some issue with the "paintings" mod.
Hard to say what the problem is without the logs.

Maybe?
* You don't have the mod installed
* You have the wrong version of the mod
* Your configuration of the mod is different to the server

Ask whoever runs the server or the paintings mod author for help if you can't figure it out.
